integration-tests/src/test/resources/environment-configuration.yml

```diff
@@ -59,9 +61,13 @@ zosmfServiceConfiguration:
     port: 10013
     serviceId: mockzosmf
 auxiliaryUserList:
-    value: 'servicesinfo-authorized,USER,validPassword;servicesinfo-unauthorized,USER1,validPassword'
+    value: 'servicesinfo-authorized,USER,validPassword,APIMTST;servicesinfo-unauthorized,USER1,validPassword,APIMTST'
```

Comment:
At what place is this used? It should be username/password and 3rd parameter here is a bit confusing

Reply:
I'll review this change, the idea was that it should be enough to have the extra user (in ITs it's used `USER` and `APIMTST` by default in some places) but for some reason I had to add it here too. I just don't remember why.

Reply:
Ok, now I remember, it's using the same Credentials object for this and for the `credentials` block on the top of the file, but it shouldn't be here as it's not using it. I will remove the extra user from this `auxiliaryUserList` and set `clientUser` to `null` in `AuxiliaryUserList.java`